docs/source/upgrade_to_newest_version.md

```diff
@@ -2,19 +2,19 @@
 
 In this topic we'll cover recommendations for upgrading to the newest release from the previous release as well as from the most recent long term support (LTS) release.
 
-## Upgrading from 2.0 to 2.1
+## Upgrading from 2.1 to 2.2
 
-The 2.1 release of Fabric is a stabilization release, featuring bug fixes and other forms of code hardening. As such there are no particular considerations needed for getting from 2.0 to 2.1, and no new capability levels to update your channels to.
+The 2.1 release of Fabric is a stabilization release, featuring bug fixes and other forms of code hardening. As such there are no particular considerations needed for getting from 2.0 to 2.1, and no new capability levels requiring particular image versions or a channel configuration update.
 
-## Upgrading to 2.1 from the 1.4.x long term support release
+## Upgrading to 2.2 from the 1.4.x long term support release
 
-Before attempting to upgrade from v1.4.x to v2.x, make sure to consider the following:
+Before attempting to upgrade from v1.4.x to v2.2, make sure to consider the following:
```

Comment:
This one can remain at v2.x so that we don't have to keep updating the doc.

Reply:
I changed this because my understanding of our plan was that we would keep this version of the doc for people going from 1.4.x to 2.2 and then any future upgrade doc would be from 2.2 to 2.3 or 2.4 and so on.

Reply:
ok, sounds good.